pkg/pool-manager/pool.go

```diff
@@ -26,10 +26,11 @@ import (
 )
 
 const (
-	RangeStartEnv            = "RANGE_START"
-	RangeEndEvn              = "RANGE_END"
-	networksAnnotation       = "k8s.v1.cni.cncf.io/networks"
-	networksStatusAnnotation = "k8s.v1.cni.cncf.io/networks-status"
+	RangeStartEnv              = "RANGE_START"
+	RangeEndEvn                = "RANGE_END"
+	networksAnnotation         = "k8s.v1.cni.cncf.io/networks"
+	networksStatusAnnotation   = "k8s.v1.cni.cncf.io/networks-status"
+	RuntimeObjectFinalizerName = "Finalizer"
```

Comment:
please change the finilizer name to something related to kubemacpool.

nit: also move the variable before `networksAnnotation`